GVFS/GVFS.FunctionalTests/GVFS.FunctionalTests.csproj

```diff
@@ -32,6 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\GVFS.Common\GVFS.Common.csproj" />

```

Comment:
What is included from `GVFS.Common.csproj`? The functional tests historically have not taken a dependency on product code.

Reply:
I think I accidentally imported it once while I was trying to look at something, and even though I removed it from `HealthTests.cs` it looks like it ended up staying there. I looked at the `git diff` this time to try and make sure I didn't accidentally push anything which I meant to remove but apparently I missed this. After you're done reviewing I'll push a commit removing this dependency here